### PR TITLE
revision: add note about automatic github articact

### PR DESCRIPTION
**Pull request by gabesullice**
Mention that GitHub will automatically create a the ZIP archive on push, so that it isn't necessary to make it locally.

### CODE REVIEW

**Comment by gabesullice** [comment]
We should add a note that the workflows must be approved before they will run.

File: README.md [hunk 1]
```diff
@@ -24,11 +24,17 @@ Source code is bundled into **three critical files**. They are:
...
+If you forked this repository, every push to the `main` branch will automatically create a new artifact you can download
+from GitHub and upload to your project immediately.
+
+Or, you can create the same release artifact locally with the following steps:
 
 1. Make your modifications and run `npm run build`.
 2. Create a ZIP archive of the new or updated `dist` directory.
-3. Upload the archive via the management UI.
+   ```shell
+   rm -f deploy.zip && pushd dist && zip -r ../deploy.zip .; popd
```
Comment (zrpnr) [inline README.md:35]: Should we make an npm command that does this? I think I have it elsewhere on some projects

**Review by zrpnr — APPROVED** [review]
This was something I didn't even realize, it's very cool that we are leveraging github like that and I think having a "release" show up on their page will tie into the idea that it's what gets uploaded to applura

File: README.md (hunk 1 above)
Comment (gabesullice) [reply README.md:35]: Sounds like a good idea. Let's do that in a follow up.

**Pull request merged**